doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+.. _reconstruction_flow:
+
+==============
+Reconstruction
+==============
+
+This tutorial walks through the steps to perform reconstruction using DIPY.
+Multiple reconstruction methods are available in DIPY.
+
+You can try these methods using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+-----------------------------------
+Constrained Spherical Deconvolution
+-----------------------------------
+
+This method is mainly useful with datasets with gradient directions acquired on a
+spherical grid.
+
+The basic idea with this method is that if we could estimate the response function
+of a single fiber then we could deconvolve the measured signal and obtain the
+underlying fiber distribution.
+
+In this way, the reconstruction of the fiber orientation distribution function
+(fODF) in CSD involves two steps:
+
+* Estimation of the fiber response function
+* Use the response function to reconstruct the fODF
+
+We are using ``stanford_hardi`` dataset. You can also use your own data.
+
+Create the directories in which to save the peaks volume (e.g.: ``recons_csd_output``)
+and mask file (e.g.: ``stanford_hardi_mask``)::
+
+    mkdir recons_csd_output stanford_hardi_mask
+
+The workflow for the CSD reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+arguments can also be provided. In this case, we will specify FA threshold for
+calculating the response function (``fa_thr``), spherical harmonics order used
+in the CSA fit (``sh_order``), whether to use parallelization in peak-finding
+during the calibration procedure or not (``parallel``), and the output
+directory (``out_dir``).
+
+To get the mask file, we will run the following command::
```

Comment:
To get the mask file, we will use the median Otsu thresholding method by calling the {double_backticks}dipy_median_otsu{double_backticks} command::

Reply:
Reworded in 4ce067c